app/models/User.js

```diff
@@ -1057,6 +1058,14 @@ module.exports = (User = (function () {
       return value
     }
 
+    getHomePageExperimentValue () {
```

Comment:
this is okay. but why do we add so many abtest functions in user model instead of using experimentName in usage ?

Reply:
I think it's safer to use functions than strings. Even if we were using the strings, we should use constants for them to avoid typo issues.